### ISSUE TITLE
Walk TX 10-week challenge service page

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#2288]
Draft lives here: https://docs.google.com/document/d/1_7GiFicELWsiEAt8P8ug8NIV_bOAXND2P-sd89uUYgQ/edit?usp=sharing

**Comment by desigonz** [comment]
@annieszot Would you mind taking first review? Link above!

**Comment by annieszot** [comment]
YES @desigonz ! This speaks to my former public health educator days!! On it right now.

**Comment by annieszot** [comment]
@desigonz I'm blocked by cognitive dissonance on this page. 
-How are we responding to the microsite (Healthy Places, Healthy People)? 
-I suggest, since this content will live on austin.gov, that we adhere to our style guide, but how do we stay consistent with _and_ justify our style guide when the microsite exists? 
-If Healthy Places, Healthy People has content on the Walk Texas! 10-week challenge, why not create a service page on how APH offers opportunities (services) to start/maintain an active living lifestyle?

**Comment by annieszot** [comment]
@desigonz Suggested edits added :)

**Comment by desigonz** [comment]
@annieszot Thanks for your edits and your great questions! This page was a stumper and your input is much appreciated.

I was reviewing a pretty old draft that was barebones and linked to some austintexas.gov pages. Then I compared against the info on austintexas.gov and Healthy Places Healthy People. The best, most complete information actually lives in the Word docs you can download on HPHP 🙃So, in efforts to transition content from austintexas.gov, and thinking about how we eventually want to move everything from the microsite but currently don't have all the features we need, I thought we could:
- include an overview of the various steps for this program, but link to the Word docs and forms on HPHP for now
- add the forms and docs to our long-term backlog (I just went in and added comments to that effect). For the docs, we probably would want to work with the HPHP folks to revise them and make sure we're using consistent language.
Does that make sense? Any thoughts or feedback?

Re: Walk TX vs. Walk Texas—I'm not sure that Chicago applies here, since you could consider it a brand name. Ultimately I just want to be consistent so that residents can recognize they are the same thing.

@LauraTrujillo any additional thoughts? Also, adding you for second review.

**Comment by LauraTrujillo** [comment]
Reviewed!
I have a lot of thoughts on this page, and this may be the most controversial- what if this becomes an information page and we draft it using what's in the Word docs? @desigonz if this works for you, feel free to assign it to me and I can draft it. 
Re: Walk TX vs Walk Texas- the email they use spells out Texas... so should that guide our decision at all?

**Comment by desigonz** [comment]
@LauraTrujillo assigned to you!

**Comment by annieszot** [comment]
Terrific, and I stand by the next steps (see bullets in last comment). No further feedback. Thanks @desigonz !
